torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -158,6 +158,21 @@ AOTITorchError aoti_torch_empty_strided(
   });
 }
 
+AOTI_TORCH_EXPORT AOTITorchError aoti_torch_empty_strided_cpu(
```

Comment:
Instead of making a new API, what if we add a branch in the existing ```aoti_torch_empty_strided``` function to handle cpu case, e.g.:

```
if (device_type == c10::DeviceType::CPU) {
    at::Tensor* new_tensor = new at::Tensor(at::detail::empty_strided_cpu...
}
```

I guess the performance might be comparable to the standalone version?

Reply:
why should we compile to something with runtime cost, however tiny, when we're running on CPU and thus all those costs are on the critical path?

Reply:
actually, it is kind of a pain in the code generator. I'll give it a shot this way

Reply:
Hmm, I think the cpu branch is going to be in the shim_common.cpp,  i.e. not via codegen. Or, are you referring to something else? Thanks!

Reply:
I meant that this diff causes some pain in the code generator (see above), so I'll try your way

Reply:
Ah, I see. Thanks.